irods-vfs-impl/src/main/java/org/irods/nfsrods/vfs/IRODSVirtualFileSystem.java

```diff
@@ -111,6 +115,15 @@
     private final ReadWriteAclWhitelist readWriteAclWhitelist_;
     private final boolean allowOverwriteOfExistingFiles_;
     private final boolean usingOracleDB_;
+    
+    // Maintains a set of locks for parallel writes for each (user, path)
+    // tuple. The locks are used to synchronize calls to open and close across
+    // parallel writes to the same replica. This is unique to NFSRODS because
+    // it cannot know how the client app (e.g. dd) will attempt to write data.
+    // Ultimately, the locks are used to avoid issues around replica access
+    // tokens (e.g. a call to open fails due to not having the required to
```

Comment:
remove an extra 'to' at the end.

Reply:
removed entire comment due to pending jargon bug fixes.

see https://github.com/DICE-UNC/jargon/pull/397